packages/perseus/src/strings.ts

```diff
@@ -226,18 +226,10 @@ export type PerseusStrings = {
         point2X: string;
         point2Y: string;
     }) => string;
-    srAngleSideAtCoordinates: ({
-        point,
-        side,
-        x,
-        y,
-    }: {
-        point: number;
-        side: string;
-        x: string;
-        y: string;
-    }) => string;
-    srAngleVertexAtCoordinatesWithAngleMeasure: ({
+    srAngleStartingSide: ({x, y}: {x: string; y: string}) => string;
+    srAngleEndingSide: ({x, y}: {x: string; y: string}) => string;
+    srAngleVertex: ({x, y}: {x: string; y: string}) => string;
+    srAngleVertexWithAngleMeasure: ({
```

Comment:
I updated this to be multiple different strings for each point, so that passing in the `side` substring does cause errors for translations.